mmdet/core/bbox/assigners/atss_cost_assigner.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from ..builder import BBOX_ASSIGNERS
+from ..iou_calculators import build_iou_calculator
+from .assign_result import AssignResult
+from .base_assigner import BaseAssigner
+
+
+@BBOX_ASSIGNERS.register_module()
+class ATSSCostAssigner(BaseAssigner):
```

Comment:
Please refer to https://github.com/open-mmlab/mmdetection/blob/master/tests/test_utils/test_assigner.py#L385, and write the unit test of ATSCostAssigner in it.